script.js

```diff
@@ -29,3 +29,148 @@ form.addEventListener('submit', (event) => {
     form.action = 'https://formspree.io/f/mbjwpgvg';
   }
 });
+
+// popup section
+const btn = document.querySelector('.button');
+const popupactive = document.querySelector('.popup');
+const popupConstainer = document.querySelector('.popup-container');
+
+btn.addEventListener('click', () => {
+  popupactive.classList.toggle('active');
+  popupConstainer.classList.toggle('show');
+});
+
+const projects = [
+  {
+    title: 'Tonic',
+    roles: { company: 'Canopy', position: ['Back End Dev', '2015'] },
+    image: 'images/SnapshootPortfolioD1.png',
+    alt: 'Canopy project screenshot',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript'],
+    skillsModal: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    livelink: { link: 'http://felix45.github.io/portfolio', text: 'See live' },
+    seesource: { link: 'http://github.com/felix45/portfolio', text: 'See Source' },
+    seeproject: 'See Project',
+    htmlClass: { article: ['grid-item', 'card'], cardImage: ['card-image'], cardDescription: ['card-description'] },
+    cssStyle: 'cardContainer',
+  },
+
+  {
+    title: 'Multi-Post Stories',
+    roles: { company: 'Facebook', position: ['Full Stack Dev', '2015'] },
+    image: '/images/SnapshootPortfolio.png',
+    alt: 'Multi-Post stories project screenshot',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    skillsModal: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    livelink: { link: 'https://felix45.github.io/portfolio', text: 'See live' },
+    seesource: { link: 'https://github.com/felix45/portfolio', text: 'See Source' },
+    seeproject: 'See Project',
+    htmlClass: { article: ['grid-item', 'card', 'hide-ruby'], cardImage: ['card-image', 'flex-item-2'], cardDescription: ['card-description', 'flex-item-1'] },
+    cssStyle: 'cardContainerRight',
+  },
```

Comment:
- [ ] Hi. in your app only the image of the first project is working the rest of the images are broken. this is probably because of the `/` at the beginning of the other image paths. So to fix them kindly remove the `/` at the beginning of the path
**imp** 
![image](https://user-images.githubusercontent.com/37901469/169627056-fc41e0a1-6c52-45c1-b865-d01fd2fd86b2.png)

- [ ] Also please kindly fix the size of your image in the desktop popup
**imp**
![image](https://user-images.githubusercontent.com/37901469/169627100-d32dad12-f58c-4a06-8f31-fb93c0168ded.png)